samples/document-snippets/family.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const Bigtable = require('@google-cloud/bigtable');
+const bigtable = new Bigtable();
+
+const snippets = {
+  createColmFamily: (instanceId, tableId, familyId, callback) => {
+    const instance = bigtable.instance(instanceId);
+    const table = instance.table(tableId);
+    const family = table.family(familyId);
+    // [START bigtable_create_family]
+    family
+      .create()
+      .then(result => {
+        const family = result[0];
+        // let apiResponse = result[1];
+        callback(null, family);
```

Comment:
Will this line be shown in the docs? If so, it could be hard to follow that we're calling the promise-style API, but passing the results back to a `callback` that doesn't seem to be defined.

Reply:
@stephenplusplus 
Yes this line will be shown in docs. 
You are correct the callback in docs wont make clear understanding
'll update the code with callback outside the region-tags